src/test/java/com/saucelabs/ci/BrowserFactoryTest.java

```diff
@@ -7,6 +7,7 @@
 import org.junit.Test;
 
 import java.net.URL;
+import java.nio.charset.Charset;
```

Comment:
Is there a reason you don't use java.nio.charset.StandardCharsets here and for the related UTF_8 lines below?

Reply:
@yfangsl , `java.nio.charset.StandardCharsets` is available starting from Java 7, but this module is built with Java 6 compatibility: https://github.com/saucelabs/ci-sauce/blob/master/pom.xml#L84-L85.